jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Server.java

```diff
@@ -474,6 +474,11 @@ public void setDumpBeforeStop(boolean dumpBeforeStop)
         _dumpBeforeStop = dumpBeforeStop;
     }
 
+    /**
+     * @return A {@link HttpField} instance efficiently recording the current time to a second resolution,
+     * that cannot be cleared from a {@link ResponseHttpFields} instance.
```

Comment:
Does this mean that `HttpConfiguration.setSendDateField(boolean)` is no longer applied to the response?

Reply:
No.  This is-the field applied to the response if `HttpConfiguration.setSendDateField(true)` is called.
The field cannot be cleared, but it can be set to a different value.